stackslib/src/chainstate/burn/operations/leader_key_register.rs

```diff
@@ -135,33 +135,23 @@ impl LeaderKeyRegisterOp {
         let num_inputs = tx.num_signers();
         let num_outputs = tx.num_recipients();
 
-        if num_inputs == 0 {
-            test_debug!(
+        if num_inputs == 0 || num_outputs < 1 {
```

Comment:
Why not just have `num_inputs == 0`?

Reply:
You mean num_outputs == 0? This was just copying the existing condition.

Reply:
oh sorry I misread